sonar-python-plugin/src/test/java/org/sonar/plugins/python/PythonHighlighterTest.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * SonarQube Python Plugin
+ * Copyright (C) 2011-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.plugins.python;
+
+import com.google.common.base.Charsets;
+import java.io.File;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+import org.sonar.api.batch.fs.internal.DefaultInputFile;
+import org.sonar.api.batch.fs.internal.FileMetadata;
+import org.sonar.api.batch.sensor.highlighting.TypeOfText;
+import org.sonar.api.batch.sensor.internal.SensorContextTester;
+import org.sonar.python.PythonAstScanner;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+public class PythonHighlighterTest {
+  
+  private SensorContextTester context;
+  
+  private File file; 
+  
+  @Before
+  @SuppressWarnings("unchecked")
+  public void scanFile() {
+    String dir = "src/test/resources/org/sonar/plugins/python";
+    
+    file = new File(dir + "/highlighting.py");
```

Comment:
@ivandalbosco you can use another constructor of `File`: `new File(dir, "highlighting.py")`. It's less error-prone